src/Data/Bin/Properties.agda

```diff
@@ -150,7 +150,7 @@ toℕ-pred 2[1+ x ] =  cong ℕ.pred $ sym $ ℕₚ.*-distribˡ-+ 2 1 (toℕ x)
 toℕ-pred 1+[2 x ] =  toℕ-double x
 
 toℕ-fromℕ : toℕ ∘ fromℕ ≗ id
-toℕ-fromℕ 0         = refl
+toℕ-fromℕ 0      = refl
```

Comment:
Why have you misaligned this `=` sign?

Reply:
I fix it now as usual, by taking the file with  ``emacs -nw`` and fixing the alignment. I hope it works. Does it?

Reply:
Yes. Thanks.